core/domain/question_domain_test.py

```diff
@@ -339,10 +339,7 @@ def test_strict_validation_passes(self):
         """Test to verify validate method of a finalized Question domain object
         with correct input.
         """
-        try:
```

Comment:
Why was this changed? Was it because ValidationError is already raised in `validate()`?

Reply:
self.question is valid here that's why no exception will be raised here

Reply:
Ok